traffic-sniffer/lib/request-metadata/sniffer-manager/mock-manager-controller.ts

```diff
@@ -0,0 +1,180 @@
+import { Express, NextFunction, Request, Response } from "express";
+import { SnifferManager } from "./sniffer-manager";
+import { MockNotFoundError } from "../sniffer/mock/exceptions";
+
+export class MockManagerController {
```

Comment:
Should we add a validation layer going ahead to check if the inputs are valid or not as such to avoid unnecessary 5xxs??

Reply:
Yes we will add zod validations for all the backend. With custom errors!

Reply:
https://github.com/idodav/sharkio/issues/87
Added an issue for this